pandas/_testing/__init__.py

```diff
@@ -195,7 +195,7 @@
     ]
 ]
 
-if not pa_version_under1p01:
+if not pa_version_under5p0:
```

Comment:
This confuses me a bit. Does this mean that pyarrow is not installed at all? If minimum is 5, we can’t have anything below it

Reply:
I think based on `pandas/compat/pyarrow.py`, the `not pa_version_under_x` means pyarrow is not installed or pyarrow is above version `x`.

So for these lines these objects are not created if pyarrow is not installed or pyarrow e.g. version 3.

Reply:
Thx, was just confused about the under since that shouldn’t be a valid option anymore